micrometer-core/src/main/java/io/micrometer/core/instrument/logging/LoggingMeterRegistry.java

```diff
@@ -66,7 +66,11 @@ public LoggingMeterRegistry() {
     }
 
     public LoggingMeterRegistry(LoggingRegistryConfig config, Clock clock) {
-        this(config, clock, new NamedThreadFactory("logging-metrics-publisher"), log::info, null);
+        this(config, clock, log::info);
+    }
+
+    public LoggingMeterRegistry(LoggingRegistryConfig config, Clock clock, Consumer<String> loggingSink) {
```

Comment:
I think the broadest ctor (the one below this) should have all the possible parameters.

Reply:
Also, can you add some tests?